scripts/common/monitoring.sh

```diff
@@ -0,0 +1,10 @@
+
+function maybe_apply_prometheus_monitor() {
+    if [ -n "$PROMETHEUS_VERSION" ]; then
+        if [ -n "$ROOK_VERSION" ]; then
+            echo "Applying Prometheus service monitor custom resource"
+            curl "https://raw.githubusercontent.com/rook/rook/v${ROOK_VERSION}/deploy/examples/monitoring/service-monitor.yaml" \
```

Comment:
We cant do this since we have to take into account airgap. instead you should add this to generate.sh and save in the add-on bundle. Also you should write this as a patch not apply it directly. I would follow [this](https://github.com/replicatedhq/kURL/blob/66491a41f4b6b1919e4f10c9fd8c73d1bb0078eb/addons/kotsadm/1.86.0/install.sh#L41-L43) pattern. That being said this code should be moved to the Rook add-on.

Reply:
Good point. I would package it with prometheus.